### ISSUE TITLE
Codehilite doesn't work inside of footnotes

### ISSUE THREAD

**Issue by startling** [Python-Markdown/markdown#73]
```
Neither fenced code nor codehilite work inside of footnotes.[^example]

[^example]: some example code (which doesn't work):

    ~~~~{.bash}
    pwd
    cd ~/desktop
    ~~~~


Ordinary code blocks (with indents) will be parsed as code but not highlighted.[^example_two]

[^example_two]: some more code, this time indicated with tabs.

        #!python
        print 'quux'


Thanks!
```

**Comment by waylan** [comment]
First of all fenced code blocks don't work at all unless they are at the document root. This is a know limitation of that extension which is being addressed by issue #53. However, any other properly implemented extension should work. I can see that at least Codehilite is not, which is weird because other blocklevel extensions (def_list) work fine.

Thanks for the report. I've updated the title to better reflect the actual problem.

**Comment by startling** [comment]
Thank you!
